homeassistant/components/roku/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      The integration only has an entity service, registered in the media_player platform.
+  appropriate-polling:
+    status: done
+    comment: |
+      This integration uses 10 seconds as its default polling rate. Apps and Channels are fetched every 15 minutes.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
I believe you don't have to do `dataclasses.replace` anymore to clone a discovery info